launcher/sdw_notify/NotifyApp.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+"""
+Notification dialog that appears when user has not applied security updates
+recently. Prompts user to check for updates or defer reminder.
+"""
+from enum import Enum
+from sdw_util import Util
+from sdw_notify import strings
+
+if Util.get_qt_version() == 5:
+    print("Using Qt5 (experimental)")
+    from PyQt5.QtWidgets import QMessageBox
+else:
+    from PyQt4.QtGui import QMessageBox
+
+
+class NotifyStatus(Enum):
+    """
+    Supported exit states from Notify dialog.
+    """
+
+    DEFER_UPDATES = "0"
+    CHECK_UPDATES = "1"
```

Comment:
@gonzalo-bulnes I thought about the random strings/longer ints, but because I am handling QMessageBox's own return values (which can only be `QMessageBox.Ok` or `QMessageBox.No` and returning basically a flag, I think the risk of the wrong thing being returned is minimal.

As for why they are strings, it was for stylistic consistency with the existing enum structure in `Updater.py`, but I am open to changing it and/or changing both at the same time (again to keep consistency).

Reply:
No strong feelings, as you say, they're flags and I too think we'd notice any (unlikely) misuse during review. The type is easy to change anytime if we want to, so I wouldn't overthink it. If we find a strong argument in the future, it will be easy to do whatever makes sense then 👍